src/locales/locale-en.json

```diff
@@ -154,6 +155,11 @@
     "USERPROFILE": "Visit your profile",
     "CHANGE_GROUP": "Change group"
   },
+  "SIDENAV": {
+    "INFO_SUB": "How to build your foodsaving group",
+    "BLOG_SUB": "All news around karrot.world",
```

Comment:
Seems a bit weird to have references to `karrot.world` inside our code base, especially when we want to make karrot more customizable. It was there before, but now it will also get translated...

Reply:
True. I'll change it.

Reply:
At some point, we need to tackle the issue of site customizations. My last proposal in https://github.com/yunity/karrot-frontend/issues/831#issuecomment-363807645 didn't receive much feedback, but it's clearly not ideal (for example it's not easily translatable).
Or maybe it's not an issue right now and can be tackled in a more distant future. Then we would just add the things to our repo, even though they are specific to karrot.world.